Kconfig.projbuild

```diff
@@ -106,8 +106,147 @@ config AUTOCONNECT_WIFI
     bool "Autoconnect WiFi on boot"
     default "n"
     depends on AUTOSTART_ARDUINO
+    depends on ARDUINO_SELECTIVE_WiFi
```

Comment:
I do not think that I use `AUTOCONNECT_WIFI` in code... do i? And why the option in general depend on setting that might not even be available if `ARDUINO_SELECTIVE_COMPILATION` is off. Or am I not getting something?

Reply:
Autoconnect WIFI was there in Kconfig, so I left it as is. The point of dependency there is to ensure You will not autoconnect WIFI if it's not compiled in. In default setting that specific key is enabled (default = all arduino goes in). Indeed, it's never used, so maybe just remove it so that people will not rely on it?